gramps/plugins/lib/libgedcom.py

```diff
@@ -1562,10 +1558,15 @@ def __ansel_to_unicode(self, s):
         return ans
 
     def __init__(self, ifile, __add_msg):
-        BaseReader.__init__(self, ifile, "", __add_msg)
+        BaseReader.__init__(self, ifile, "ANSEL", __add_msg)
+        # In theory, we should have been able to skip the encode/decode from
```

Comment:
I don't understand this comment. It's not Python 2; you _must_ encode/decode for I/O.